tests/PHPUnit/System/expected/test_ImportLogs__PagePerformance.get_month.xml

```diff
@@ -6,5 +6,5 @@
 	<avg_time_dom_processing>0</avg_time_dom_processing>
 	<avg_time_dom_completion>0</avg_time_dom_completion>
 	<avg_time_on_load>0</avg_time_on_load>
-	<avg_page_load_time>0</avg_page_load_time>
+	<avg_page_load_time>1.67</avg_page_load_time>
```

Comment:
This change actually lets me wonder if our calculation for the avg page load time is correct. Why should the page load time be a lot higher then the avg server time in this case?

Actually I think we might maybe also change the calculation of the `page_load_hits`
```
$selects[] = sprintf('(SUM(%s)/%s) as page_load_hits', implode(' + ', $hitsColumns), count($hitsColumns));
```
That actually means that a visit only fully counts if all fields are filled. This works if normally all fields have a value I guess, but as here in the test only one column is filled, causing the avg page load time to be much higher than it should be. I guess we should count a hit as soon as one field is given instead.

Reply:
@sgiehl yes, I was wondering the same, not sure that will work, just a draft. 
```php
        $selects[] = "count(idlink_va) as page_load_hits";
        $query = $this->getLogAggregator()->queryActionsByDimension([], 'having page_load_total>0', $selects, false, null, $joinLogActionOnColumn);

```